packages/wdio-shared-store-service/src/server.js

```diff
@@ -0,0 +1,38 @@
+const polka = require('polka')
+const { json } = require('body-parser')
+
+const store = {}
+
+const app = polka()
```

Comment:
why do we need a web server if we can just store the data in memory?

Reply:
I don't know how to do it, to be honest. 
How can I share data between two child process? Don't want use process events for that

Reply:
You can launch the storage part as a single identity in the launcher. The launcher is run only once in the launcher process. In the launcher you can then add the port to the wdio config which gets propagated into all child processes.

For data transportation between storage and child processes you are right that an simple server seems to be the simplest solution but there might be also other options.

Reply:
The port added to config is not propagated to child processes, sorry to say.
That was something I've tried first

Reply:
Yes, for capabilities it works